Sources/Compiler/AST/PassingConvention.swift

```diff
@@ -16,6 +16,7 @@ public enum PassingConvention: Codable {
   /// Yielded.
   case yielded
 
+  /// Creates a convention matching the given remote type capability.
```

Comment:
What does it mean to create a convention matching a remote type capability?   A doc comment should give me enough information to write a test.

Also, please don't use "the given _\<type rewritten in english\>_."  It's a silly convention used by DevPubs that takes up valuable words that could be used to give real salient information.

```suggestion
  /// Creates an instance matching `capability`.
```

But then use the space you've won to write something meaningful :-)

Reply:
I think I am getting unconvinced that all functions must be 100% understandable to the point of someone being able to write unit tests by their single-line documentation. That is a fine principle, really, but here I don't think any comment can be clearer than the 6-lines body.

I'll be honest. Have to scratch my head trying to come up with a good comment for this kind of function feels like a disincentive to refactor the boilerplate that it removed elsewhere.

Reply:
Don't view scratching your head to come up with a good comment as a burden; it's an opportunity. It is part of the process of understanding your own code and the problem you're solving, and eventually arriving at the right API.  The comment is for the consumer of the function who doesn't see the implementation, not the person who wrote it.  I don't know what a “remote type capability” is.  Should there even be a type for it distinct from `PassingConvention`?

Reply:
Perhaps, although I am not sure what should be done for `yielded`. The current code predates the latest iteration of remote types.

I'll keep the status quo until remote types are better specified.